pkgs/data/machine-learning/mnist/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenvNoCC, fetchurl }:
+let
+  srcs = {
+    train-images = fetchurl {
+      url = "http://yann.lecun.com/exdb/mnist/train-images-idx3-ubyte.gz";
+      sha256 = "029na81z5a1c9l1a8472dgshami6f2iixs3m2ji6ym6cffzwl3s4";
+    };
+    train-labels= fetchurl {
```

```suggestion
    train-labels = fetchurl {
```
(nitpick) for consistency with the rest of nixpkgs
Idem for the next 2 ones